src/components/ImportForm/GitImportActions.tsx

```diff
@@ -19,6 +33,7 @@ const GitImportActions: React.FunctionComponent<GitImportActionsProps> = ({
     isSubmitting,
     isValidating,
     setErrors,
+    handleReset,
```

Comment:
You can take the `handleSubmit` method from formikProps as well. No need to pass it down from parent.

Reply:
Updated.